src/components/Answers.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const Answers = (props) => {
+  const { id, answers } = props;
+
+  return(
+    <article className="answers">
+      <input
```

Comment:
We likely only want to allow users to select a single answer per question. The radio element can take a `name` attribute that will limit our selection. See example [here](http://www.w3schools.com/html/tryit.asp?filename=tryhtml_input_radio).